composer.json

```diff
@@ -25,20 +25,20 @@
         "php":                      "^7.0",
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
-        "sebastian/exporter":       "^1.0 || ^2.0",
-        "symfony/console":          "^2.7 || ^3.0",
-        "symfony/event-dispatcher": "^2.7 || ^3.0",
-        "symfony/process":          "^2.7 || ^3.0",
-        "symfony/finder":           "^2.7 || ^3.0",
-        "symfony/yaml":             "^2.7 || ^3.0",
+        "sebastian/exporter":       "^2.0",
+        "symfony/console":          "^3.2",
```

Comment:
Do these changes bring any improvements (like removing code to support both 2.7 and 3.x, or we require 3.x because of useful features that we want to use but they exist only in 3.x)?

If I correctly understand how composer version constraints work this will forbid using future versions of `phpspec` with symfony 2.7 (which is LTS and will be maintained until mid 2018).

I'm not sure if updates just for the sake of updates worth it.